modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -24,160 +30,169 @@ const addButtons = (editor: Editor, selections: Selections, selectionTargets: Se
 
   const cmd = (command: string) => () => editor.execCommand(command);
 
-  editor.ui.registry.addButton('tableprops', {
+  // TODO TINY-8172: unwind this before merging the feature branch
+  const addButtonIfRegistered = (name: string, spec: AddButtonSpec) => {
+    if (editor.editorCommands.hasCustomCommand(spec.command)) {
```

Comment:
Due to changes in #7572 please change all of these to `queryCommandSupported`
```suggestion
    if (editor.editorCommands.queryCommandSupported(spec.command)) {
```

Reply:
I have done this as part of #7559